stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -394,6 +579,154 @@ public final class _ExecutionContext {
   }
 }
 
+// Elements in `outputs` can come from two sources:
+// a) Symbolic tensors produced by tensor ops, and added as trace graph nodes.
+// b) Concrete tensors produced by host code (e.g. Tensor(1.0)).
+func finalizeAndExecuteTraceFn(fnName: String,
+                               inputs: [Tensor<Float>],
+                               outputs: [CTensorHandle]) -> [CTensorHandle] {
+  guard let tracedFn = _RuntimeConfig.traceState.getFunction else {
+    fatalError("Not in tracing mode!.")
+  }
+  _RuntimeConfig.traceState = .notTracing
+
+  return tracedFn.finalizeAndExecute(traceeBasicName: fnName,
+                                     inputs: inputs,
+                                     outputs: outputs)
+}
+
+public extension TensorArrayProtocol {
+  // The returned handles are owned by the caller.
+  func _getCTensorHandles() -> [CTensorHandle] {
```

Comment:
Avoid using "get" and use a computed property when the method does not take any arguments.
```suggestion
  var cTensorHandles: [CTensorHandle] {
```

Reply:
Done